project/dbmanager/views.py

```diff
@@ -13,16 +13,13 @@
 knowledgeManager = KnowledgeManager()
 
 
-def split_links(links):
-    return links.split(';')
-
-
 @dbmanager_blueprint.route('/')
 @token_required
 def index():
     all_data = knowledgeManager.get_all_data()
     for item in all_data:
-       item['splitedLinks'] = split_links(item['links'])
+        item['splitedLinks'] = item['links'].split(';')
```

Comment:
@alexx97-python  Does it work well? 
When we have no links and ```item['links']``` returns ```null``` or whatever

Reply:
@Proladge It works perfect as for me, doesn't cause any error, just leave the block empty without links